docs/resources/metrics_policy.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: "wavefront"
```

Comment:
Would it make sense to show metrics policy as a data source in an example?  I am curious how that would work.

Reply:
Added